editor.html

```diff
@@ -373,8 +373,10 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                                 <li><span title="Spanish"><a lang="es" href="#" class="action lang-choice" id="es" tabindex="122">Español</a></span></li>
                                 <li><span title="Polish"><a lang="pl" href="#" class="action lang-choice" id="pl" tabindex="123">Polski</a></span></li>
                                 <li><span title="Croatian"><a lang="hr" href="#" class="action lang-choice" id="hr" tabindex="124">Hrvatski</a></span></li>
-                            </ul>
-                            <div><a id="add-language-link" class="action" title="Add a language translation to the Editor" href="https://support.microbit.org/support/solutions/articles/19000106022-translating-the-micro-bit-python-editor" target="_blank" tabindex="124">Add a language</a></div>
+                                <li><span title="Chinese (simplified)"><a lang="zh-CH" href="#" class="action lang-choice" id="zh" tabindex="125">简体中文</a></span></li>
+                                <li><span title="Chinese (traditional)"><a lang="hk" href="#" class="action lang-choice" id="hk" tabindex="126">繁体中文</a></span></li>
+                            </ul>crow
```

Comment:
Not sure how a crow got in??

Reply:
🤣
Every once in a while I still find things like "git status" in the middle of my code.